hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/expiry/ExpirySystem.java

```diff
@@ -236,11 +236,6 @@ public final void extendExpiryTime(Data dataKey, long now, long lastUpdateTime)
             return;
         }
 
-        Map<Data, ExpiryMetadata> expireTimeByKey = getOrCreateExpireTimeByKeyMap(false);
-        if (isEmpty()) {
-            return;
-        }
-
         ExpiryMetadata expiryMetadata = getExpiryMetadataForExpiryCheck(dataKey, expireTimeByKey);
```

Comment:
How do we assert here that `expireTimeByKey` won't be null?

Reply:
`isEmpty()` checks it.

Reply:
sorry for inconvenience, I missed the above occurrence.